### DISCUSSION TITLE
Can't toggle visibility on model parts

### DISCUSSION THREAD

**Discussion by danilokrulj** [google/model-viewer#3991] [Q&A]
Hi guys, so I used to toggle certain model parts visibility on and off in some project, when I tried changing the model viewer module from 1.9.0 to the current one in that project it seems to brake the code. When I stop the code a few lines before the visibility line is executed I get the value and can change it, however, when I get past that line of code it tells me property is undefined. Any clues what's going on?

Screenshot from devtools console window:
![image](https://user-images.githubusercontent.com/92090684/206193923-9b421316-209e-4605-99bf-ff63e0d50df4.png)

**Comment by elalish** [comment]
This is why we say "there be dragons" when you leave our public API. I'm not sure off the top of my head; maybe we lost or gained a parenting level?

**Reply by danilokrulj** [reply]
For some reason I had to add "._model" after the "modelViewer[sceneSym]" but it seems to have fixed the issue. Thank you though :)
